pladder/bjbot.py

```diff
@@ -0,0 +1,31 @@
+import os
+from pladder.plugin import Plugin
+
+class BjBotPlugin(Plugin):
+    def __init__(self, bot):
+        super().__init__()
+        self._datorbas = dict()
+        self.importera(self._datorbas, os.path.join(os.path.dirname(__file__), "triggers",
```

Comment:
Can you change this into getting the path like this:

```
os.path.join(bot.statedir, "triggers.txt")
```

The state directory will be set up by the bot to always exist (and it will be backuped in the future). Also, when the python package is installed, the state dir will not be along the source files.